js/renovation/ui/scroll_view/scrollable_native.tsx

```diff
@@ -894,7 +892,9 @@ export class ScrollableNative extends JSXComponent<ScrollableNativePropsType>()
       return false;
     }
 
-    this.update();
+    if (!this.props.updateManually) {
```

Comment:
Condition seems to be incorrect:

the 'validate' method should do validation without additional actions, additional actions should be out of this method
tryGetAllowedDirection/isScrollingOutOfBound are called after 'update' is completed and their result can be changed if 'update' was not called.

Reply:
Can we remove it?
hg/mobile/rev/c52851107602

Reply:
Can we remove updateManually?
hg/mobile/rev/c52851107602

Reply:
fixed in c4f2bdc